src/overview.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: page
+title: Overview
+---
+
+# Overview
+
+Workbox is a collection of libraries and build tools that make it easy to
+store your website's resources locally, on your users' devices. Consider Workbox
+if you want to:
+
+* Make your site work offline.
+* Improve load performance on repeat-visits. You can use Workbox to store
```

Comment:
I'd say this is fine as is. HTTP cache can be wiped out whenever and in that case this will import load perf on repeat visits.

I'd keep this intro as simple to follow and highlevel as possible.